code/modules/recycling/sortingmachinery.dm

```diff
@@ -185,6 +188,15 @@
 			wrapped_item.AddComponent(/datum/component/pricetag, sticker.payments_acc, sticker.cut_multiplier)
 		update_appearance()
 
+	if(istype(item, /obj/item/boxcutter))
+		if( (item.icon_state == "boxcutter_on"))
+			unwrap_contents(user)
+			post_unwrap_contents(user, rip_open = FALSE)
+			return
+		else
+			balloon_alert(user, span_warning("The boxcutter isn't active!"))
```

Comment:
```suggestion
			balloon_alert(user, span_warning("open [item] first!"))
```

Reply:
this is wrong because it's a balloon alert and balloon alerts should not have spans and should be kept as small as possible

Reply:
problem is 'open it first' or something makes you question if it means the boxcutter or the wrapper

Reply:
The suggestion I made was to change it to "prime the boxcutter", you wouldn't tell someone to "open" a switchblade anyway